src/app/store/models/mail.model.ts

```diff
@@ -7,3 +7,13 @@ export interface Mail {
   send?: boolean;
   mailbox: number;
 }
+export interface Mailboxes {
```

Comment:
Please rename it to `Mailbox`
its a single mail box, its array will be mailboxes